### DISCUSSION TITLE
ZSH_THEME does not take effect There will be no change to the theme

### DISCUSSION THREAD

**Discussion by Yunak** [ohmyzsh/ohmyzsh#11277] [Help] [answered]
ZSH_THEME does not take effect There will be no change to the theme
archlinux
![image](https://user-images.githubusercontent.com/16012667/196879758-78a08da8-2c88-4563-9879-ae4f0ea8ce24.png)

**Comment by mcornella** [comment, marked as answer]
Try `omz reload` instead of `source .zshrc`, and if that doesn't work post your zshrc file. It's possible that another theme is not able to be overwritten.

**Reply by Yunak** [reply]
Thanks has been resolved to remove this discussion  https://github.com/ohmyzsh/ohmyzsh/discussions/10916#discussioncomment-2965803
